### DISCUSSION TITLE
Is Mono-project Repo currently down?

### DISCUSSION THREAD

**Discussion by jchen351** [mono/mono#21765] [Q&A]
I tried to click on few items on https://download.mono-project.com/repo/ubuntu/pool/main/m/index.html but get 
this error
```
<Error>
<Code>BlobNotFound</Code>
<Message>The specified blob does not exist. RequestId:95e11924-501e-006f-4669-a21ae0000000 Time:2024-05-09T23:35:57.0589048Z</Message>
</Error>
```

**Comment by akoeplinger** [comment]
Yes this was a problem due to misconfiguration on our end, see https://github.com/mono/mono/issues/21767
